lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -68,15 +68,20 @@ sub test_network_interface {
     # Configure the network interface to use DHCP configuration
     my $nic = script_output "ssh root\@$guest \"grep '$mac' /sys/class/net/*/address | cut -d'/' -f5 | head -n1\"";
     assert_script_run("ssh root\@$guest \"echo BOOTPROTO=\\'dhcp\\' > /etc/sysconfig/network/ifcfg-$nic\"");
-    if ($guest =~ m/sles11/i) {
+    if (!get_var("SRIOV_PCI_PASSSHTROUGH")) {
+      if ($guest =~ m/sles11/i) {
         assert_script_run("ssh root\@$guest service network restart", 90);
-    } else {
+      } else {
         assert_script_run("ssh root\@$guest systemctl restart wickedd wickedd-dhcp4 wicked", 90);
+      }
     }
     assert_script_run("ssh root\@$guest ifup $nic", 90);
     my $addr = script_output "ssh root\@$guest ip -o -4 addr list $nic | awk \"{print \\\$4}\" | cut -d/ -f1";
 
     # Route our test via the tested interface
+    if ((script_run "ssh root\@$addr 'ip r | grep -q $target'") == 0) {
```

Comment:
"== 0" is not needed.

Reply:
yes, change to use 'wc' to judge if $target exists.